mobile/src/test/java/org/openhab/habdroid/util/UtilTest.java

```diff
@@ -42,16 +44,31 @@ public void normalizeUrl() throws Exception {
 
     @Test
     public void parseSitemapList() throws Exception {
-        List<OpenHABSitemap> sitemapList = Util.parseSitemapList(createSitemapDocument());
+        // openHAB 1
+        List<OpenHABSitemap> sitemapList = Util.parseSitemapList(createSitemapOH1Document());
         assertFalse(sitemapList.isEmpty());
 
         // Should be sorted, null first
         assertEquals(null, sitemapList.get(0).getLabel());
         assertEquals("Garden", sitemapList.get(1).getLabel());
         assertEquals(8, sitemapList.size());
+
+        // openHAB 2
+        sitemapList = Util.parseSitemapList(createJsonArray(1));
+        assertFalse(sitemapList.isEmpty());
+
+        assertEquals("Main Menu", sitemapList.get(0).getLabel());
+        assertEquals(1, sitemapList.size());
+
+        sitemapList = Util.parseSitemapList(createJsonArray(2));
+        assertFalse(sitemapList.isEmpty());
+
+        assertEquals("Main Menu", sitemapList.get(0).getLabel());
+        assertEquals("HOME", sitemapList.get(1).getLabel());
+        assertEquals(2, sitemapList.size());
     }
 
-    private Document createSitemapDocument() throws ParserConfigurationException, IOException, SAXException {
+    private Document createSitemapOH1Document() throws ParserConfigurationException, IOException, SAXException {
```

Comment:
if you rename it, I would love to rename it to "getSitemapOH1Document", it's a getter as it returns something :)